spec/controllers/items_controller_spec.rb

```diff
@@ -80,41 +72,37 @@
           expect { post_create(name: "name", wishlist_id: wishlist.id) }.to change(Item, :count).by(1)
         end
 
-        include_examples "redirects to the wishlist page"
+        include_examples "redirects to", :wishlist
 
-        it "shows a notice" do
-          expect(flash[:notice]).not_to be_nil
-        end
+        include_examples "shows a flash message", :notice
       end
 
       context "with invalid parameters" do
-        before { post_create(name: nil, wishlist_id: wishlist.id) }
+        subject! { post_create(name: nil, wishlist_id: wishlist.id) }
```

Comment:
So, like a before, but you keep the value returned? OK.
![](http://i.imgur.com/dVDJiez.gif)